llvm/include/llvm/Frontend/OpenMP/OMP.td

```diff
@@ -1179,6 +1179,13 @@ def OMP_Tile : Directive<"tile"> {
   let association = AS_Loop;
   let category = CA_Executable;
 }
+def OMP_Stripe : Directive<"stripe"> {
+  let allowedOnceClauses = [
+    VersionedClause<OMPC_Sizes, 51>,
```

Comment:
I have set it to `51` but I think it should be `60`. If you think so too then it might be a good idea to create another PR just for that because it might break a bunch of LIT tests. WDYT?

Reply:
Yes, should be 60

Reply:
I see. In this case I think it would be better to create a PR for making this change first (I noticed a lots of LIT errors when I tried changing this value to 60). and then continue with this PR. WDYT?

Reply:
Sounds good